lib/experimental/editor-settings.php

```diff
@@ -37,6 +37,10 @@ function gutenberg_enable_experiments() {
 	if ( $gutenberg_experiments && array_key_exists( 'gutenberg-zoom-out-experiment', $gutenberg_experiments ) ) {
 		wp_add_inline_script( 'wp-block-editor', 'window.__experimentalEnableZoomOutExperiment = true', 'before' );
 	}
+
+	if ( $gutenberg_experiments && array_key_exists( 'gutenberg-simple-editing-mode', $gutenberg_experiments ) ) {
+		wp_add_inline_script( 'wp-block-editor', 'window.__experimentalSimpleEditingMode = true', 'before' );
```

Comment:
Personally I don't fell like we need an experiment if we take over the "select mode" and improve it.

Reply:
If we take over "Select Mode" do we need `blockEditingMode` at all, or can we just conditionalise all the block tools (in both locations) so that they 

- have the right visual appearance.
- show only the necessary controls (i.e. respect the `role=content` attribute setting in block json)

Reply:
I think `blockEditingMode` probably needs to stay for BC reasons but I think ideally the block adapts to the tool without needing to call `setBlockEditingMode` explicitly.